crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
My comment has been lost, but again I insist depends is not needed. It's not a computed field, it's a method.